app/models/post.rb

```diff
@@ -13,10 +13,31 @@
 class Post < ApplicationRecord
   belongs_to :user
   scope :recent, -> { order('created_at desc') }
+  after_create :send_to_action_cable
 
   def self.all_for_user(user)
     Post.where(user_id: user.id)
         .or(Post.where(user_id: user.friend_ids))
         .or(Post.where(user_id: user.user_ids))
   end
+
+  private
+
+  def send_to_action_cable
+    data = { message: to_html, action: 'new_post' }
+
+    user.friend_ids.each do |friend_id|
```

Comment:
refactor to a single line with `{...}` and create a new method for the repeated content

Reply:
Done!